db/migrate/20230815113802_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :users, null: false, foreign_key: true
```

Comment:
As per the ER diagram provided to you, the relation between the like and the user is set up through the author just like for the posts. Kindly update your code just like you did for the posts.